crates/lib3h/tests/integration_test.rs

```diff
@@ -266,6 +266,44 @@ fn launch_three_memory_nodes_test(test_fn: ThreeNodesTestFn, can_setup: bool) ->
     Ok(())
 }
 
+#[test]
+fn test_mirror_suite() {
+    enable_logging_for_test(true);
+    for (test_fn, can_setup) in MIRROR_TEST_FNS.iter() {
+        launch_mirror_test(*test_fn, *can_setup).unwrap();
+    }
+}
+
+// Do general test with config
+fn launch_mirror_test(test_fn: MultiNodeTestFn, can_setup: bool) -> Result<(), ()> {
+    let test_fn_ptr = test_fn as *mut std::os::raw::c_void;
+    println!("");
+    print_test_name("IN-MEMORY MIRROR TEST: ", test_fn_ptr);
+    println!("==========================");
```

Comment:
Change printlns to debug/trace?

Reply:
there are a bunch more of these and it's just the tests so I'm ok leaving this for now.